e2e_samples/parking_sensors/adf/linkedService/Ls_KeyVault_01.json

```diff
@@ -5,7 +5,7 @@
 		"annotations": [],
 		"type": "AzureKeyVault",
 		"typeProperties": {
-			"baseUrl": "https://mdwdokvdevkdnam.vault.azure.net/"
+			"baseUrl": "https://mdwdops-kv-dev-16.vault.azure.net/"
```

Comment:
Instead of actual names like "mdwdops-kv-dev-16" which is a bit confusing (what is "dev-16"), can we put placeholder values like:
"baseUrl": "https://<storageaccountname>.vault.azure.net/"

I believe this gets replace anyway.

Reply:
We don't need to have them. Its just when we develop and test the sample locally, ADF saves these files to the repo automatically. So basically, anytime you want to dev/test the sample the placeholders will get overwritten to the dev/test values. Changing the values by hand then checking in, feels error prone...

Ideally we need some sort of automatic, consistent way that the dev/test values get overridden back into the original placeholders safely when we check in. I'll have a look at git hooks to see if this can be done.